test/services/uri_service_test.rb

```diff
@@ -2,10 +2,12 @@
 
 class UriServiceTest < ActiveSupport::TestCase
   describe 'URI service utility' do
-    it 'instantiates a uri from string' do
-      uri = UriService.new("some yolo test uri").uri
-      assert uri.is_a?(URI)
-    end
+    ## this is an invalid URL
+    # it 'instantiates a uri from string' do
```

Comment:
What do you propose we do with this test? Is it no longer needed and deleteable, should we re-work it, or?